crates/krun/Cargo.toml

```diff
@@ -13,10 +13,13 @@ anyhow = { version = "1.0.82", default-features = false, features = ["std"] }
 bpaf = { version = "0.9.11", default-features = false, features = [] }
 byteorder = { version = "1.5.0", default-features = false, features = ["std"] }
 env_logger = { version = "0.11.3", default-features = false, features = ["auto-color", "humantime", "unstable-kv"] }
+futures-util = "0.3.30"
```

Comment:
Could we add `default-features = false, features = [...]` for all?

Reply:
Done, thanks!